cwltool/main.py

```diff
@@ -162,7 +162,16 @@ def arg_parser():  # type: () -> argparse.ArgumentParser
                         help="Will be passed to `docker run` as the '--net' "
                              "parameter. Implies '--enable-net'.")
 
-    parser.add_argument("--on-error", type=str,
+    exgroup = parser.add_mutually_exclusive_group()
+    exgroup.add_argument("--enable-tool-registry", action="store_true", help="Enable resolution using tool registry",
+                        dest="enable_tool_registry", default=True)
+    exgroup.add_argument("--disable-tool-registry", action="store_false", help="Disable resolution using registry",
+                        dest="enable_tool_registry", default=True)
+
+    parser.add_argument("--add-tool-registry", action="append", help="Add a tool registry to use for resolution, default %s" % tool_registries,
+                        dest="tool_registries", default=[])
```

Comment:
The help text for all three of theses needs to mention what kind of tool registry; in this case it is "a GA4GH compliant tool registry"

See https://github.com/ga4gh/tool-registry-schemas/issues/17 for a discussion on this